### ISSUE TITLE
refund with Widget

### ISSUE THREAD

**Issue by 7c** [paymentwall/paymentwall-node#29]
Hi,

we are using Widget api to accept payments... after the ipn we upgrade users (digital goods)... All well but i am still looking a way to refund clients via API. I understand that brick API might have something available but we are using Widget API. Can you please push us towards right API ? Many thanks

**Comment by 7c** [comment]
I have got response from support, saying that we need to use https://docs.paymentwall.com/apis#section-tools-cancellation for this purpose, which obviously creates a ticket for the backend team to process.

**Comment by prashantbd** [comment]
Yes, that's correct. Cancellation API does help in issuing refunds. Although they are not instant but it is a recommended way we should be integrating it.
